mllib/src/main/scala/org/apache/spark/ml/fpm/PrefixSpan.scala

```diff
@@ -167,3 +170,27 @@ final class PrefixSpan(@Since("2.4.0") override val uid: String) extends Params
   override def copy(extra: ParamMap): PrefixSpan = defaultCopy(extra)
 
 }
+
+private[spark] class PrefixSpanWrapper(override val uid: String)
```

Comment:
`PrefixSpan` is not a transformer or estimator, add such wrapper so that it can be treated as a transformer on spark connect